src/4d.js

```diff
@@ -20,6 +20,7 @@ var Mode4D = (function (scope) {
 		this.rightMesh = null;
 
 		this.gridIsVisible = true;
+		this.useQHull = true;
```

Comment:
@OmarShehata My thoughts is that the QHull server is a fallback solution, not the default, so this would be set to false by default. It would be a good way to also limit unnecessary work for your server (though I'm sure it can handle it).

Reply:
I agree, it _should_ be a fallback. The only reason I had it as the default right now was because technically not having it on produced incorrect results. 

I could just have it off by default and let Humke know when he's teaching to turn it on for more accurate/faster results.

Reply:
how are the results technically incorrect when run on the client?

Reply:
Sorry, I should have posted this in the original description. So here's a comparison of Qhull vs client algorithm. Notice how it's missing a facet on this tesseract (we generally have an issue with coplanar points in the algorithm - if you wiggle just one of the points there it'll fix it)

![explanation](https://i.imgur.com/kSNvBGJ.gif)